readthedocs/doc_builder/python_environments.py

```diff
@@ -174,23 +174,22 @@ def setup_base(self):
             shutil.rmtree(version_path)
         self.build_env.run(
             'conda',
+            'env',
             'create',
-            '--yes',
             '--name',
             self.version.slug,
-            'python={python_version}'.format(python_version=self.config.python_version),
+            '--file',
+            self.config.conda_file,
             bin_path=None,  # Don't use conda bin that doesn't exist yet
         )
 
     def install_core_requirements(self):
-        conda_env_path = os.path.join(self.project.doc_path, 'conda')
 
         # Use conda for requirements it packages
         requirements = [
             'sphinx==1.3.1',
             'Pygments==2.0.2',
-            'docutils==0.11',
-            'mock==1.0.1',
+            'docutils',
```

Comment:
Sorry to keep nitpicking on this, but if you are concerned about disruption then should the `mock` constraint simply be left as is?